plugins/json-logger/README.md

```diff
@@ -0,0 +1,27 @@
+# @amplication/plugin-<name>
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-basic)](https://www.npmjs.com/package/@amplication/plugin-auth-basic)
```

Comment:
Change `auth-basic` in these links to the name of the plugin to point the badge to the